IdentityCore/src/MSIDError.m

```diff
@@ -124,6 +124,10 @@ MSIDErrorCode MSIDErrorCodeForOAuthErrorWithSubErrorCode(NSString *oauthError, M
     {   //Account Transfter, when user skips the QR code page.
         return MSIDErrorUserCancel;
     }
+    if (oauthError && [oauthError caseInsensitiveCompare:@"server_error"] == NSOrderedSame && [subError caseInsensitiveCompare:@"server_error"] == NSOrderedSame)
```

Comment:
Should it be handled inside MSIDErrorCodeForOAuthError?

Reply:
I understand why we can handle this error in MSIDErrorCodeForOAuthError